CONTRIBUTING.md

```diff
@@ -17,19 +17,26 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 This is the repository for Azure Monitor Workbook Templates used in the Azure Portal. Please ensure that you are opening issues in the right repository.
 
 Other repos you might be looking for:
+* [Azure Monitor Community](https://github.com/microsoft/AzureMonitorCommunity#azure-monitor-community) repo, with query, alerts, workbook samples.
 * [Log Analytics Query Examples](https://github.com/MicrosoftDocs/LogAnalyticsExamples) repo - contains example log analytics queries
 * [Azure Sentinel](https://github.com/azure/azure-sentinel) repo - contains queries, dashboards, templates used by Azure Sentinel
 
 ## Recommended setup for contributing
-- Get contributor access
-
-  In order to contribute, you'll need contributor access to the repo in order to push your branch and create a PR. If you are a Microsoft employee, mail `azmonworkbooks`  to be added as a contributor. If you are not a Microsoft employee, the quickest way is to [create a new issue][new-issue] and ask for contributor access.
+- In order to contribute, you'll need contributor access to the repo in order to push your branch and create a PR. 
+    - If you are a Microsoft employee:
+      1) Ensure your github user is already a member of the Microsoft organization 
```

Comment:
in the case they are a microsoft employee and not part of the microsoft org, they can join through this link: https://repos.opensource.microsoft.com/orgs